### ISSUE TITLE
Doesn't work fine for 'iphone x'

### ISSUE THREAD

**Issue by alexanderLanars** [willmcpo/body-scroll-lock#129]
For see this problem you need:
1. Get Iphone X, open https://bodyscrolllock.now.sh/
2. Little scroll page to top, for hide bottom panel with navigation buttons
3. Slick 'disable body scroll'
4. Swipe page in the same place, where was bottom panel with nav buttons
5. As you can see, page still scrolled

**Comment by hacknug** [comment]
You mean it doesn't stay at the same place when you force Safari to show its bars?

**Comment by alexanderLanars** [comment]
No, I mean, when you hide Safari bottom bar and touch and swipe on that place (where was bar), you can scroll page.

**Comment by hacknug** [comment]
Can't reproduce on iOS 13. It only covers some content that was visible when the bars were hidden.

**Comment by KayLeung** [comment]
I ran into this problem.

This is only happening on Safari. Because when you swipe from the bottom, it calling the bottom bar out. (On fast swiping, the body will scroll) Bad iOS behaviors!

**Comment by hacknug** [comment]
Just managed to trigger scroll in both directions when double tapping the top/bottom of the viewport. This was on the latest iOS 13 beta.

**Comment by diachedelic** [comment]
This is an old article concerning the "special" status of page areas where the menu bars sit: https://benfrain.com/the-ios-safari-menu-bar-is-hostile-to-web-apps-discuss/
